docs/sources/administration/grafana-advisor/_index.md

```diff
@@ -0,0 +1,127 @@
+weight: 300
+labels:
+  products:
+    - oss
+    - cloud
+    - enterprise
+  stage: experimental
+keywords:
+  - grafana
+  - grafana advisor
+  - monitoring
+  - instance health
+---
+
+# Grafana Advisor
+
+{{< docs/experimental product="Grafana Advisor" featureFlag="grafanaAdvisor" >}}
+
+## Overview
+
+Grafana Advisor is a monitoring tool that helps administrators keep their Grafana instances running smoothly and securely. It automatically performs regular health checks on your Grafana server, providing actionable insights and recommendations for maintaining optimal system performance.
+
+{{< admonition type="note" >}}
+Currently, Grafana Advisor only performs regular health checks on data sources, plugins, and your SSO settings, but we're planning to expand its capabilities in future releases to cover more aspects of your Grafana environment.
+
+You can suggest new checks and provide feedback through this [form](https://docs.google.com/forms/d/e/1FAIpQLSf8T-xMZauFXZ1uHw09OjZLT_AaiY-cl-hJGwC6Krkj0ThmZQ/viewform).
+{{< /admonition >}}
+
+## Before you begin
+
+To set up Grafana Advisor you need:
+
+- Administration rights in your Grafana organization.
+- If you're running Grafana on-premise, enable the required feature toggles in your Grafana instance. Refer to [Enable required feature toggles](#enable-feature-toggles) for instructions. This is not required if you're using Grafana Cloud, as the feature toggles are enabled by default.
+
+### Enable feature toggles
+
+To activate Grafana Advisor, you need to enable the `grafanaAdvisor` feature toggle. This will automatically install the Grafana Advisor application to your server if it is not already installed. For additional information about feature toggles, refer to [Configure feature toggles](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/feature-toggles/).
+
+To enable the required feature toggles, add them to your Grafana configuration file:
+
+1. Open your Grafana configuration file, either `grafana.ini` or `custom.ini`. For file location based on the operating system, refer to Configuration file location.
+1. Locate or add a `[feature_toggles]` section. Add this value:
+
+   ```ini
+   [feature_toggles]
+   grafanaAdvisor = true
+   ```
+
+1. Save the changes to the file and restart Grafana.
+
+## Access Grafana Advisor
+
+1. Log in to your Grafana instance with an administrator account
+1. Navigate to the Administration section
+1. Select "Advisor" from the navigation menu
+
+![<Grafana Advisor UI>](/media/docs/grafana-advisor/grafana-advisor-ui.png)
+
+## Understand the Advisor interface
+
+### Action needed section
+
+This collapsible section displays issues requiring immediate attention:
+
+- Each issue is listed with descriptive text outlining the problem
+- For data source issues, Grafana Advisor displays the specific data source name.
+- A "Fix me" button appears. This button points you to the relevant issues to fix
```

Comment:
The text and the buttons will change, depending on the issue type. I'd change this line with something like: 

```
 - For each item, Grafana Advisor displays the specific name of the item that needs to be fixed.
 - One or more buttons appear. These buttons point you to different links to fix the issue, retry the check or hide the error.
```